openssl/openssl_test.go

```diff
@@ -28,3 +28,14 @@ func TestMain(m *testing.M) {
 	openssl.CheckLeaks()
 	os.Exit(status)
 }
+
+func TestCheckVersion(t *testing.T) {
+	v := os.Getenv("GO_OPENSSL_VERSION_OVERRIDE")
```

Comment:
Am I right that this envvar should always set when running tests? Perhaps it makes sense to log something if `v` is ""?

Reply:
Good point. I've improved `TestMain` so it tries to find an available OpenSSL version in case `GO_OPENSSL_VERSION_OVERRIDE` is not set.